docs/intro/scala.md

```diff
@@ -90,7 +90,7 @@ env OPENAI_TOKEN=<your-token> sbt -J--enable-preview <your-command>
 <details>
 <summary>IntelliJ</summary>
 
-- Set the Java version to at least 19
+- Set the Java version to at least 20
 - Set VM options to `--enable-preview`
```

Comment:
Do we still need `--enable-preview` if we set the default to 20?

Reply:
Good question, let me check it

Reply:
According to the [Java documentation](https://openjdk.org/jeps/437), the `StructuredTaskScope` API still resides in the incubator module, so the `--enable-preview` flag is still required.